packages/dashboard-frontend/src/store/DevfileRegistries/selectors.ts

```diff
@@ -63,6 +67,31 @@ export const selectMetadataFiltered = createSelector(
   },
 );
 
+export const selectEmptyWorkspaceUrl = createSelector(
+  selectState,
+  selectRegistriesMetadata,
+  (state, metadata) => {
+    const v2Metadata = filterDevfileV2Metadata(metadata);
+    const emptyWorkspaceMetadata = v2Metadata.find(meta => meta.tags.includes(EMPTY_WORKSPACE_TAG));
+    return emptyWorkspaceMetadata?.links?.v2;
+  },
+);
+
+export const selectDefaultDevfile = createSelector(
+  selectState,
+  selectEmptyWorkspaceUrl,
+  (state, devfileLocation) => {
+    if (!devfileLocation) {
+      return undefined;
+    }
+    const devfileContent = state.devfiles[devfileLocation]?.content;
+    if (devfileContent) {
+      return load(devfileContent) as devfileApi.Devfile;
```

Comment:
I think you need to wrap this line with a try-catch.

Reply:
Ok. I fixed it.